### PR TITLE
Update DEPLOYING.md

### PR DESCRIPTION
**Pull request by zalenski**
@pettermahlen

### CODE REVIEW

File: DEPLOYING.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Once you've got that in place, you should be able to do deployment using the fol
 
 ```
 # deploy snapshot version
-mvn clean deploy
+mvn clean deploy # -Prelease to test signing
 
-# make and deploy a relase
+# make and deploy a release
```
Comment (zalenski) [inline DEPLOYING.md]: spelling

File: DEPLOYING.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ Once you've got that in place, you should be able to do deployment using the fol
 
 ```
 # deploy snapshot version
-mvn clean deploy
+mvn clean deploy # -Prelease to test signing
```
Comment (zalenski) [inline DEPLOYING.md]: -Prelease to test signing?

File: DEPLOYING.md [hunk 3]
```diff
@@ -13,9 +13,9 @@ Once you've got that in place, you should be able to do deployment using the fol
...
 
-# make and deploy a relase
+# make and deploy a release
 mvn release:clean release:prepare -Prelease
-mvn release:perform
+mvn release:perform -Prelease
```
Comment (zalenski) [inline DEPLOYING.md]: needed?

Comment (rouzwawi) [inline DEPLOYING.md]: This activates the staging plugin, so probably needed.

**Comment by pettermahlen** [comment]
:+1:

**Comment by pettermahlen** [comment]
I believed that we should also move the release plugin config outside the 'release' profile, so you wouldn't have to specify ```-Prelease``` at all. But from what I can tell, the ```releaseProfiles``` setting is only applied to the 'perform' goal. That might mean that -Prelease isn't necessary for 'perform', but I think better safe than sorry..

**Comment by rouzwawi** [comment]
I'm ok with moving it out if it doesn't effect normal builds. `mvn verify`

**Comment by pettermahlen** [comment]
I wasn't clear in my previous comment. What I meant was, "I used to think it was a good idea to move it out, but I've changed my mind". So I think this update to the instructions is good, and we don't need to make other changes.

**Comment by rouzwawi** [comment]
@pettermahlen ah I misread

**Pull request merged**